settings/aboutpage.md

```diff
@@ -75,60 +75,8 @@ The first image depicts the normal white setting (off), the second image shows i
 
 - **Disable card expansion animations**: Disables the animations when expanding a widget card.
 - **Disable page transition animations**: Disables the transitions when switching from one page to the other.
-
-### Enable Web Audio sink support
-
-- This setting is also part of the miscellaneous settings.
```

Comment:
From here on:

I have the feeling you deleted most of what I have written here. Please don't as I have put quite an amount of work by providing context, background and examples into it.

You move around redundant information but don't drop it, thanks.